opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -45,6 +45,8 @@
 #include <opm/input/eclipse/Parser/ParserKeywords/O.hpp>
 #include <opm/input/eclipse/Parser/ParserKeywords/P.hpp>
 #include <opm/input/eclipse/Parser/ParserKeywords/T.hpp>
+#include <opm/input/eclipse/Parser/ParserKeywords/X.hpp>
+#include <opm/input/eclipse/Parser/ParserKeywords/Y.hpp>
 
```

Comment:
Why is this needed?

Reply:
They are needed for XMF and YMF.  Should not be in this PR.